pkg/ccl/benchccl/rttanalysisccl/rttanalysisccl_test: TestBenchmarkExpectation failed

pkg/ccl/benchccl/rttanalysisccl/rttanalysisccl_test.TestBenchmarkExpectation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21272936?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21272936?buildTab=artifacts#/) on release-26.1 @ [e1bb04a035b18f2ed7033b9c2bfa7292387aa14d](https://github.com/cockroachdb/cockroach/commits/e1bb04a035b18f2ed7033b9c2bfa7292387aa14d):


```
=== RUN   TestBenchmarkExpectation
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/7418df5abfb13dbfaf6c0beee25f240a/logTestBenchmarkExpectation1226280000
    test_log_scope.go:82: use -show-logs to present logs inline
    validate_benchmark_data.go:120: success: expected AlterSurvivalGoals/alter_empty_database_from_region_to_zone to perform KV lookups in [12, 12], got 13
    validate_benchmark_data.go:120: success: expected AlterSurvivalGoals/alter_empty_database_from_zone_to_region to perform KV lookups in [12, 12], got 13
    validate_benchmark_data.go:120: success: expected AlterRegions/alter_empty_database_drop_region to perform KV lookups in [12, 16], got 15
    validate_benchmark_data.go:120: success: expected AlterTableLocality/alter_from_global_to_rbr to perform KV lookups in [18, 18], got 19
    validate_benchmark_data.go:120: success: expected AlterTableLocality/alter_from_rbr_to_global to perform KV lookups in [10, 10], got 11
    validate_benchmark_data.go:120: success: expected AlterSurvivalGoals/alter_populated_database_from_zone_to_region to perform KV lookups in [22, 22], got 23
    validate_benchmark_data.go:120: success: expected AlterSurvivalGoals/alter_populated_database_from_region_to_zone to perform KV lookups in [22, 22], got 23
    validate_benchmark_data.go:120: success: expected AlterRegions/alter_populated_database_add_region to perform KV lookups in [11, 11], got 12
    validate_benchmark_data.go:120: success: expected AlterTableLocality/alter_from_global_to_regional_by_table to perform KV lookups in [13, 13], got 14
    validate_benchmark_data.go:120: success: expected AlterTableLocality/alter_from_rbr_to_regional_by_table to perform KV lookups in [10, 10], got 11
    validate_benchmark_data.go:120: success: expected AlterTableLocality/alter_from_regional_by_table_to_rbr to perform KV lookups in [18, 18], got 19
    validate_benchmark_data.go:120: success: expected VirtualTableQueries/select_from_crdb_internal.zones_(10_tables) to perform KV lookups in [10, 10], got 10
    validate_benchmark_data.go:120: success: expected AlterRegions/alter_empty_database_add_region to perform KV lookups in [11, 11], got 12
    validate_benchmark_data.go:117: fail: expected AlterRegions/alter_populated_database_drop_region to perform KV lookups in [14, 14], got 16
    validate_benchmark_data.go:120: success: expected AlterPrimaryRegion/alter_empty_database_alter_primary_region to perform KV lookups in [12, 14], got 13
    validate_benchmark_data.go:120: success: expected AlterPrimaryRegion/alter_populated_database_alter_primary_region to perform KV lookups in [12, 14], got 13
    validate_benchmark_data.go:120: success: expected VirtualTableQueries/select_from_crdb_internal.zones_(50_tables) to perform KV lookups in [10, 10], got 10
    validate_benchmark_data.go:120: success: expected AlterPrimaryRegion/alter_empty_database_set_initial_primary_region to perform KV lookups in [13, 13], got 14
    validate_benchmark_data.go:120: success: expected AlterTableLocality/alter_from_regional_by_table_to_global to perform KV lookups in [13, 13], got 14
    validate_benchmark_data.go:120: success: expected AlterPrimaryRegion/alter_populated_database_set_initial_primary_region to perform KV lookups in [14, 14], got 15
    validate_benchmark_data.go:120: success: expected VirtualTableQueries/select_from_crdb_internal.zones_(100_tables) to perform KV lookups in [10, 10], got 10
    validate_benchmark_data.go:73: see the -rewrite flag to re-run the benchmarks and adjust the expectations
    validate_benchmark_data.go:74: usage: ./dev test --rewrite pkg/bench/rttanalysis -f=TestBenchmarkExpectation/ --test-args '-rewrite-iterations=N'
    validate_benchmark_data.go:107: -- test log scope end --
--- FAIL: TestBenchmarkExpectation (143.46s)
=== RUN   TestBenchmarkExpectation/AlterRegions
--- FAIL: TestBenchmarkExpectation/AlterRegions (83.15s)
=== RUN   TestBenchmarkExpectation/AlterRegions/alter_populated_database_drop_region
    test_server_shim.go:94: cluster virtualization disabled due to issue: #156308 (expected label: C-test-failure)
    rtt_analysis_bench.go:224: AlterRegions/alter_populated_database_drop_region: got 16.5, expected 14
    rtt_analysis_bench.go:230: wrote jaeger trace to /artifacts/tmp/_tmp/7418df5abfb13dbfaf6c0beee25f240a/logTestBenchmarkExpectation1226280000/AlterRegions_alter_populated_database_drop_region.jaeger.json
--- FAIL: TestBenchmarkExpectation/AlterRegions/alter_populated_database_drop_region (83.15s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBenchmarkExpectation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-61745

--- a/pkg/bench/rttanalysis/testdata/benchmark_expectations
+++ b/pkg/bench/rttanalysis/testdata/benchmark_expectations
@@ -9,7 +9,7 @@ exp,benchmark
 11,AlterRegions/alter_empty_database_add_region
 12-16,AlterRegions/alter_empty_database_drop_region
 12-13,AlterRegions/alter_populated_database_add_region
-15-16,AlterRegions/alter_populated_database_drop_region
+14-16,AlterRegions/alter_populated_database_drop_region
 12,AlterRole/alter_role_with_1_option
 15,AlterRole/alter_role_with_2_options
 19,AlterRole/alter_role_with_3_options
@@ -36,12 +36,12 @@ exp,benchmark
 13,AlterTableDropConstraint/alter_table_drop_1_check_constraint
 13,AlterTableDropConstraint/alter_table_drop_2_check_constraints
 13,AlterTableDropConstraint/alter_table_drop_3_check_constraints
-28,AlterTableLocality/alter_from_global_to_rbr
+27-30,AlterTableLocality/alter_from_global_to_rbr
 9-10,AlterTableLocality/alter_from_global_to_regional_by_table
 21-24,AlterTableLocality/alter_from_rbr_to_global
 21-24,AlterTableLocality/alter_from_rbr_to_regional_by_table
 9-10,AlterTableLocality/alter_from_regional_by_table_to_global
-28,AlterTableLocality/alter_from_regional_by_table_to_rbr
+27-30,AlterTableLocality/alter_from_regional_by_table_to_rbr
 8,AlterTableSplit/alter_table_split_at_1_value
 11,AlterTableSplit/alter_table_split_at_2_values
 14,AlterTableSplit/alter_table_split_at_3_values
@@ -57,9 +57,9 @@ exp,benchmark
 16,DropDatabase/drop_database_1_table
 16,DropDatabase/drop_database_2_tables
 16,DropDatabase/drop_database_3_tables
-21,DropRole/drop_1_role
-29,DropRole/drop_2_roles
-38,DropRole/drop_3_roles
+19-21,DropRole/drop_1_role
+28-30,DropRole/drop_2_roles
+37-39,DropRole/drop_3_roles
 13,DropSequence/drop_1_sequence
 15,DropSequence/drop_2_sequences
 17,DropSequence/drop_3_sequences

